social-network/public/reset.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Like everything, this is also a npm module. You may or may not find it handy on future assignments https://www.npmjs.com/package/reset-css